.pre-commit-config.yaml

```
@@ -21,11 +21,10 @@ repos:
       - id: end-of-file-fixer
       - id: trailing-whitespace
   - repo: https://github.com/psf/black-pre-commit-mirror
-    rev: 23.9.1
+    rev: 23.11.0
     hooks:
       - id: black
         language_version: python3
-        args: ['--target-version', 'py37']
   - repo: https://github.com/astral-sh/ruff-pre-commit
     # Ruff version.
     rev: v0.0.291
```

May I suggest:

```suggestion
    rev: v1.0.6
```

while at it. Ref: https://github.com/astral-sh/ruff/releases/tag/v0.1.6